src/ngraph/pass/fused_op_decomposition.hpp

```diff
@@ -25,6 +25,30 @@ namespace ngraph
 {
     namespace pass
     {
+        /// \brief  The FusedOpDecomposition pass is used to decompose a fused op
+        /// if it is not supported by the backend.
+        ///
+        /// \details The pass is run on each node in the computation graph. If the
+        /// node is a fused op and is not supported by the backend, the pass
```

Comment:
rephrasing a little bit, how about this? 
"By default, this pass decomposes the fused ops. If the backend supports a fused op, then it can provide a callback function while registering the pass. The callback function can then provide logic to prevent decomposing the supported op."

Reply:
💯